### PR TITLE
Update many additional iOS translations to use the Shared Strings Keys

### PR DESCRIPTION
**Pull request by wbaumann**
The change includes:
 - Removed duplicate and/or unused strings from the Localizable.strings
 - Updated a variety of string mappings, deleted unused settings views
 - Removed an unused category
 - Updated additional string references and remove more shared text
 - Applied additional tweaks to translations for the pdf reports
 - Fixed a few incorrect keys that were automatically mapped

### CODE REVIEW

**Comment by wbaumann** [comment]
I'm merging this, so I can keep working on this today. I'll fix anything you point out :)

File: SmartReceipts/Common/UI/AppTheme.swift [hunk 1]
```diff
@@ -10,8 +10,8 @@ import Foundation
 
 class AppTheme: NSObject {
     
-    static let appTitle = "Smart Receipts"
-    static let appTitlePlus = "Smart Receipts Plus"
+    static let appTitle = LocalizedString("sr_app_name")
+    static let appTitlePlus = LocalizedString("sr_app_name_plus")
```
Comment (EvsenevDev) [inline SmartReceipts/Common/UI/AppTheme.swift]: Thanks!

File: SmartReceipts/Persistence/Migrations/DatabaseCreateAtVersion11.m [hunk 2]
```diff
@@ -78,7 +78,6 @@ - (BOOL)insertDefaultCategoriesIntoDatabase:(Database *)database {
             LocalizedString(@"category_meals_justified", nil), LocalizedString(@"category_meals_justified_code", nil), //
             LocalizedString(@"category_stationery_stations", nil), LocalizedString(@"category_stationery_stations_code", nil), //
             LocalizedString(@"category_training_fees", nil), LocalizedString(@"category_training_fees_code", nil), //
-            LocalizedString(@"pref_distance_header", nil), LocalizedString(@"category.distance.code", nil), //
```
Comment (EvsenevDev) [inline SmartReceipts/Persistence/Migrations/DatabaseCreateAtVersion11.m]: needless category?

**Comment by EvsenevDev** [comment]
All looks good for me 👍

File: SmartReceipts/Persistence/Migrations/DatabaseCreateAtVersion11.m (hunk 2 above)
Comment (wbaumann) [reply SmartReceipts/Persistence/Migrations/DatabaseCreateAtVersion11.m]: Yep - we use it when generating the distances reports (e.g. converting distances to receipts) but not as a main category. I figured the distance page is better for this :)

**Pull request merged**